src/actions/helpers/helpers.js

```diff
@@ -6,7 +7,7 @@
 function usernameExists(username, usernameToUserInfo) {
   const validUsername =
     username === 'me' || username === 'I' || usernameToUserInfo.has(username);
-  if (!validUsername) throw Error(`the user ${username} does not exist`);
+  if (!validUsername) throw new customError(`the user ${username} does not exist`, 'EA1000')
```

Comment:
you can export the error codes as well as the map so here you can use error codes instead of the string